whisper/stt/processing/utils.py

```diff
@@ -58,20 +58,20 @@ def get_language(language = None):
     language_fields = language.split("-")
     if len(language_fields) == 2:
         language = language_fields[0]
-    language = language.lower()
     # "*" means "all languages"
     if language == "*":
         language = None
     # Convert French -> fr
     if isinstance(language, str) and language not in LANGUAGES:
-        language = {v: k for k, v in LANGUAGES.items()}.get(language)
+        candidate_language = {v: k for k, v in LANGUAGES.items()}.get(language.lower())
```

Comment:
@AudranBert note that I moved the "lower" back here.
Because I really want to be case insensitive ONLY for "human-readable" things like "French", "cantonese", ...
I DON'T want case-insensitiveness for short codes that are standard (it should be "fr", not "FR"). Case insensitiveness is not in the documentation anyway.
En général, les systèmes trop permissifs, qui interprètent trop ce que donne l'utilisateur (sans qu'il en soit forcément conscient), peuvent jouer des sales tour (un exemple tout bête sur ce cas: l'utilisateur qui veut passer "fr-CA", mais il s'est gourré dans le split de son coté et passe "CA", qui est interprété en "ca" = "catalan" ...).

Reply:
As you want, but he can also put "CA-fr" because if he don't know the order xD